src/PioneerUtils.jsm

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const { utils: Cu } = Components;
+Cu.import('resource://gre/modules/XPCOMUtils.jsm');
+
+const { TelemetryController } = Cu.import('resource://gre/modules/TelemetryController.jsm', null);
+
+Components.utils.importGlobalProperties(['crypto']);  // Crypto is not available by default
```

Comment:
This should be `Cu.importGlobalProperties`, since we've aliased `Component.utils` to `Cu` above.